rebar.lock

```diff
@@ -10,7 +10,7 @@
   0},
  {<<"aesophia">>,
   {git,"https://github.com/aeternity/aesophia.git",
-      {ref,"b61e3270f91129667d5948c4e0ce4b214a9d180c"}},
+      {ref,"2edafe0adcfea7dcfc52cbe8fe28bf7cace04bfa"}},
```

Comment:
https://github.com/aeternity/aesophia/compare/b61e3270f91129667d5948c4e0ce4b214a9d180c...2edafe0adcfea7dcfc52cbe8fe28bf7cace04bfa

Reply:
I see b61e included in https://github.com/aeternity/aesophia/commits/2edafe0adcfea7dcfc52cbe8fe28bf7cace04bfa

Reply:
Unclear if backward compatible.